test/surface/test_convex_hull.cpp

```diff
@@ -321,14 +323,13 @@ TEST (PCL, ConvexHull_2dsquare)
   input_cloud->points.resize (input_cloud->width * input_cloud->height);
   
   //rng
-  boost::mt19937 rng_alg;
-  boost::uniform_01<boost::mt19937> rng (rng_alg);
-  rng.base ().seed (12345u);
+  std::mt19937 rng(12345u);
+  std::uniform_real_distribution<float> rd (0.0f, 1.0f);
```

Comment:
Set the constructor parameters appropriately here -1.0f to 1.0f, and then remove the math logic which is employed in lines 331 and 332.